README.md

```diff
@@ -28,10 +28,25 @@ Look through the application code. If you have the old Lambda Times (Applied Jav
 
 ## Self-Study/Essay Questions
 
-- [ ] What are PropTypes used for? Please describe why it's important to type check our data in JavaScript.
-- [ ] Describe a life-cycle event in React?
-- [ ] Explain the details of a Higher Order Component?
-- [ ] What are three different ways to style components in React? Explain some of the benefits of each.
+- [x] What are PropTypes used for? Please describe why it's important to type check our data in JavaScript.
+
+Proptypes assure you are getting the correct type of data. It will help with assuring the user has a flawless experience without running into any bugs.
```

Comment:
Proptypes only give console warnings. While it is helpful, it won't block incorrect data from reaching your component. You would need something like typescript to achieve that.